### ISSUE TITLE
Starting Gold in MP Campaigns always resets to a multiple of 25

### ISSUE THREAD

**Issue by Discontinuum** [wesnoth/wesnoth#8226]
### Game and System Information

- **Version:** 1.16.*

### Description of the bug

There is no way to set an arbitrary amount of starting gold in an MP campaign, on the game creation screen it's always reset to the closest multiple of 25. So if I want 80 gold, it sets 75. If I want 90, it sets 100. 

Even Legend of Wesmere has this bug, it has 100, 120, 140 gold depending on difficulty and it's been reset

### Expected behavior

It should be possible to set custom starting gold for MP campaigns especially when it's locked and players can't change it

**Comment by gfgtdf** [comment]
i cannot reproduce this. When i try to start LoW mp i start with 120 as specified in the cfg file.

**Comment by Discontinuum** [comment]
Hm, I see this (1.16.5 from flatpak, Middle difficulty, the first chapter). 125 gold

![изображение](https://github.com/wesnoth/wesnoth/assets/68666085/cf7ab38c-50a9-4bd9-b28f-f0bf4426de7e)

**Comment by Discontinuum** [comment]
Easy difficulty, 150
![изображение](https://github.com/wesnoth/wesnoth/assets/68666085/68c0a347-aab1-40f0-8f48-10814abb24b4)

**Comment by Wedge009** [comment]
Related to #6678?

**Comment by soliton-** [comment]
I don't think so. As gfgtdf says it seems like a UI-only issue. The settings are respected as far as I can tell but it looks confusing in the UI since they aren't displayed accurately.

**Comment by gfgtdf** [comment]
that said, its problem sill effects scenarios that don't lock the gold.

Similar to  https://github.com/wesnoth/wesnoth/issues/8048, imo this applies to all those mp settings, i think The available range should not be static and instead change to make the default value that is set ion the scenario at least possible. The case of gold  is a bit more complex, but the problem is very similar.

**Comment by Wedge009** [comment]
Where the gold slider can be changed, it does indeed appear to be locked to increments of 25.
